spec/support/files/example_playbook.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Just being picky here, could you update the comments to be in English? Because we are opening this repo so I prefer everything in English. Thanks!